### PR TITLE
Prefix the title with the BF class

### LINKED ISSUE #3126: add BF-specific prefix to resource title in Editor
- For title of resource, use `label` (as currently implemented). If resource is a bf:Work, prefix with "WORK - "; if resource is a bf:Instance, prefix with "INSTANCE - "; if resource is a bf:Item, prefix with "ITEM - ".
- see #3082 for design (only the title part of the design)
- see #3082 for discussion of desirability of having BF-specific prefix; possibility for having non-BF specific prefix

### PR DESCRIPTION
**Pull request by jcoyne**
## Why was this change made?

Fixes #3126

### CODE REVIEW

**Comment by justinlittman** [comment]
I've refactored this to be a badge, per Michelle and Astrid.

image

**Review by justinlittman — APPROVED** [review]

File: src/components/editor/ResourceComponent.jsx [hunk 1]
```diff
@@ -71,10 +73,12 @@ const ResourceComponent = () => {
         <div className="row">
           <div className="col-md-10">
             <h3>
-              {resource.label}
+              <ResourceTitle resource={resource} />
               <CopyToNewButton />
               <PreviewButton />
             </h3>
+            <CopyToNewButton />
+            <PreviewButton />
```
Comment (mjgiarlo) [inline src/components/editor/ResourceComponent.jsx:81]: So we want these buttons both in the h3 and outside of it?

**Review by mjgiarlo — APPROVED** [review]
Have a couple questions, none of them blocking.

File: src/components/editor/ResourceComponent.jsx [hunk 2]
```diff
@@ -51,11 +52,12 @@ const ResourceComponent = () => {
     <div id="resourceTemplate">
       <section className="resource-header">
         <h3 id="resource-header">
-          {resource.label}
+          <ResourceTitle resource={resource} />
           <CopyToNewButton />
           <PreviewButton />
         </h3>
         <CopyToNewMessage />
+        <CopyToNewMessage />
```
[the remark this answers is not in the document]
Comment (justinlittman) [reply src/components/editor/ResourceComponent.jsx]: More specifically, merge conflict cruft. Thanks for the catch.

File: src/components/editor/ResourceComponent.jsx (hunk 1 above)
Comment (justinlittman) [reply src/components/editor/ResourceComponent.jsx]: Nope. Same problem as above. Fixed.

**Pull request merged**